### ISSUE TITLE
The php72 test-report job is always successful

### ISSUE THREAD

**Issue by gmponos** [doctrine/coding-standard#209]
Is this correct?

https://github.com/doctrine/coding-standard/pull/205/checks?check_run_id=788441821

As you can see on this job there are differences but the job is success.

Is this addressed here? https://github.com/doctrine/coding-standard/issues

**Comment by greg0ire** [comment]
It does look weird… can you try doing an obvious mistake in one of your PRs and see if it is caught by that job?

**Comment by gmponos** [comment]
Not sure exactly what you call "obvious".. check [this PR](https://github.com/doctrine/coding-standard/pull/206) where I'm adding a new sniff and many files of this repo do not comply with it.

Right from the first job you'll notice the PHP 72 that is successful.

**Comment by greg0ire** [comment]
Link for the lazy: https://github.com/doctrine/coding-standard/pull/206/checks?check_run_id=791333879

**Comment by lcobucci** [comment]
@greg0ire oh my, that's indeed a terrible mistake.

It seems to be failing since https://github.com/doctrine/coding-standard/pull/148.

I believe it should have been something like this:

```sh
if [ $$? -ne 0 ]; then
    if  [ $(PHP_74_OR_NEWER) -eq 1 ]; then
        git apply -R tests/php-compatibility.patch;
    fi

    exit 1;
fi
```

Fixed in https://github.com/doctrine/coding-standard/pull/210

**Comment by lcobucci** [comment]
@gmponos thanks for reporting this issue :+1:
